index.html

```diff
@@ -2,78 +2,144 @@
 <html>
   <head>
     <title>Coding Basics</title>
-    <style>
-      * {
-        box-sizing: border-box;
-      }
-
-      body {
-        font-family: sans-serif;
-        margin-left: 30px;
-        margin-right: 30px;
-      }
-
-      #header {
-        text-align: center;
-      }
-
-      #container {
-        background-color: pink;
-        margin: 40px auto;
-        max-width: 800px;
-        padding: 38px 31px;
-      }
-
-      input {
-        font-size: 21px;
-        line-height: 33px;
-        margin: 0 0 23px 0;
-        padding: 0 9px;
-        width: 100%;
-      }
-
-      button {
-        font-size: 21px;
-        line-height: 33px;
-        margin: 0 0 23px 0;
-        padding: 0 6px;
-      }
-
-      #output-div {
-        background-color: lightgrey;
-        margin: 20px auto;
-        padding: 20px;
-        width: 100%;
-      }
-    </style>
+    <link rel="stylesheet" href="style.css" type="text/css" />
   </head>
 
   <body>
     <h1 id="header">Basics: Beat That! 🚀</h1>
     <div id="container">
-      <p>Input:</p>
-      <input id="input-field" />
+      <table>
+        <tr>
+          <th>Number of Players:</th>
+          <th>Number of Dice:</th>
+          <th>Dice Mode:</th>
+          <th>Game Mode:</th>
+        </tr>
+        <tr>
+          <td>
+            <input id="input-player" min="2" max="8" type="number" value="2" />
+          </td>
+          <td>
+            <input id="input-dice" min="2" max="8" type="number" value="2" />
+          </td>
```

Comment:
i can still do this even though you set the max hahaha

![image](https://user-images.githubusercontent.com/40411953/156772970-47a956ae-3672-44b8-9255-9b923d6a04b1.png)

for additional challenge, look into how you can disable keyboard input or at least do some control on what the user inputs using the keyboard